slashing-protection/src/integration-test/resources/interchange/singleNullSigningRootBlock.json

```diff
@@ -0,0 +1,17 @@
+{
+  "metadata": {
+    "interchange_format_version": "5",
+    "genesis_validators_root": "0x123456"
+  },
+  "data": [
+    {
+      "pubkey": "0xb845089a1457f811bfc000588fbb4e713669be8ce060ea6be3c6ece09afc3794106c91ca73acda5e5457122d58723bed",
+      "signed_blocks": [
+        {
+          "slot": "12345"
```

Comment:
This is actually a missing signing_root. Ideally would be good to test that both signing_root with null and missing signing_root both behave the same way.

Reply:
added an extra entry which has null value for this field.